change/@uifabric-file-type-icons-2019-12-04-16-34-43-azbanduk-add-stream-icon.json

```diff
@@ -0,0 +1,8 @@
+{
+  "type": "minor",
+  "comment": "ComboBox: empty errorMessage shouldn't create white space. (#11353)",
```

Comment:
Can you update this message to describe what the change does?

Reply:
Done!